Dockerfile

```diff
@@ -16,6 +16,7 @@ ARG BLUEOCEAN_VERSION=1.5.0
 ARG JENKINS_STAGING=/usr/share/jenkins/ref/
 ARG MESOS_PLUG_HASH=5216d7ecef0bc8923ff510aec6659e2c7e7611cb
 ARG PROMETHEUS_PLUG_HASH=a347bf2c63efe59134c15b8ef83a4a1f627e3b5d
+ARG STATSD_PLUG_HASH=statsd
```

Comment:
This may not pick up future changes if that hash/tag is update on s3, but cached in docker. Do we want to keep this or use a sha1?